sdcm/nemesis.py

```diff
@@ -513,6 +513,25 @@ def disrupt_switch_between_PasswordAuthenticator_and_SaslauthdAuthenticator_and_
         else:
             raise UnsupportedNemesis("SaslauthdAuthenticator can't work without saslauthd environment")
 
+    def disrupt_toggle_internode_compression(self):
+        def toggle_internode_compression(current_compression):
```

Comment:
Don't think this is needed,  the algorithm of this nemesis is very simple
1) get 'internode_compression' from scylla.yaml -> original_val
2) remove value from ['dc', 'all', 'none'] -> values_for_selection
3) randomly selecto from values_for_selection -> desired_compression_val
4) iterate over all nodes, set the desired_compression_val, restart the node

Reply:
@bentsi The four steps you have wrote are already in the implementation.
get_internode_compression_new_value_randomly()  is a helper function to return random value different from the current value.

Multiple calls to disrupt_random_set_compression_on_all_dbnodes() can be done in one test.
Each time the internode_compression value is different according to the following cases:
- Not the first call to replace the value.
- First time with existing value in scylla.yaml.
- First time when internode compression k/v does not exist in scylla.yaml.